thrift_collector_client.go

```diff
@@ -91,7 +91,6 @@ func (client *thriftCollectorClient) ConnectClient() (Connection, error) {
 	} else {
 		transport, err := thrift.NewTHttpPostClient(client.collectorURL, client.reportTimeout)
 		if err != nil {
-			maybeLogError(err, client.verbose)
```

Comment:
Replace with an `onEvent(...)`?

Reply:
For better or worse, `ConnectClient` errors are currently getting emitted on the caller's side.

Reply:
👍